npm/ng-packs/packages/theme-shared/src/lib/components/loader-bar/loader-bar.component.ts

```diff
@@ -24,6 +24,12 @@ import { HttpWaitService, RouterWaitService, SubscriptionService } from '@abp/ng
   imports: [CommonModule],
 })
 export class LoaderBarComponent implements OnDestroy, OnInit {
+  private router = inject(Router);
```

Comment:
`router` also does not seem to be used here, so we can check and remove accordingly